### PR TITLE
Add documentation for adf option

### PR DESCRIPTION
**Pull request by hjkristenson**

### CODE REVIEW

File: hyp3_gamma/metadata/templates/insar/readme.md.txt.j2 [hunk 1]
```diff
@@ -38,6 +38,17 @@ pixel spacing = 20 * azimuth looks
...
 Products with 20x4 looks have a pixel spacing of 80 m, while those with 10x2 looks have a pixel spacing of 40 m.
 
+----------------
+### Adaptive Phase Filtering ###
+
+When ordering InSAR On Demand products, users can choose to set a custom value for the adaptive phase filter (adf). This filter improves fringe visibility and reduces phase noise in interferograms, particularly for InSAR pairs with low coherence.
+
+The filter is applied adaptively, meaning that regions with high coherence (where fringe patterns are easily discernible) will have more smoothing applied, while regions with low coherence will undergo less smoothing, so as not to remove residues that may represent actual deformation signals. Using this filter approach allows more of the interferogram to be unwrapped.
+
+Users can set the adf value within the range of 0 to 1, with 0 indicating that no filtering occurs, and 1 indicating the strongest level of filtering. The value should generally be greater than 0.2, and interferograms with very low coherence will benefit from higher values (closer to 1). Setting this value to 0 will result in no filter being applied.
```
Comment (jtherrmann) [inline hyp3_gamma/metadata/templates/insar/readme.md.txt.j2:48]:
```suggestion
Users can set the adf exponent (alpha) value within the range of 0 to 1, with 0 indicating that no filtering occurs, and 1 indicating the strongest level of filtering. The value should generally be greater than 0.2, and interferograms with very low coherence will benefit from higher values (closer to 1). Setting this value to 0 will result in no filter being applied.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Added
 - The `--phase-filter-parameter` option now accepts a value of `0.0`, which indicates that the adaptive phase filter will be skipped.
+- Documentation for the adaptive phase filter parameter options in product readme and xml files
```
Comment (jtherrmann) [inline CHANGELOG.md:13]:
```suggestion
- Documentation for the adaptive phase filter parameter in product readme and xml files.
```

File: hyp3_gamma/metadata/templates/insar/readme.md.txt.j2 [hunk 3]
```diff
@@ -38,6 +38,17 @@ pixel spacing = 20 * azimuth looks
 
 Products with 20x4 looks have a pixel spacing of 80 m, while those with 10x2 looks have a pixel spacing of 40 m.
 
+----------------
+### Adaptive Phase Filtering ###
+
+When ordering InSAR On Demand products, users can choose to set a custom value for the Goldstein-Werner adaptive phase filter (adf). This filter improves fringe visibility and reduces phase noise in interferograms, particularly for InSAR pairs with low coherence.
```
Comment (jtherrmann) [inline hyp3_gamma/metadata/templates/insar/readme.md.txt.j2:44]:
```suggestion
When ordering InSAR On Demand products, users can choose to set a custom value for the parameter used in the Goldstein-Werner adaptive phase filter (adf). This filter improves fringe visibility and reduces phase noise in interferograms, particularly for InSAR pairs with low coherence.
```

**Comment by hjkristenson** [comment]
Verified that all the trufflehog results are storymap links

**Review by jtherrmann — APPROVED** [review]

**Pull request merged**